anchor-reference-server/src/main/java/org/stellar/anchor/reference/model/Quote.java

```diff
@@ -37,7 +40,10 @@ public class Quote {
 
   String transactionId;
 
-  public static Quote of(GetRateRequest request, String price) {
+  @Convert(converter = PriceDetailListConverter.class)
```

Comment:
Basically, this tells JPA that a complex object should be stored in a database column as a string, instead of storing it in a separate table.

This was new to me too and sounded pretty cool.

Reply:
Awesome. I got to change some of the fields that I have done earlier.